docs/Homebrew-linuxbrew-core-Maintainer-Guide.md

```diff
@@ -73,8 +73,8 @@ changes that the upstream Homebrew developers have made.
 brew merge-homebrew --core
 ```
 
-Merging all the changes from upstream in one go is usually
-undesirable since our build servers will time out. Instead, attempt
+Merging all the changes from upstream in one go is risky since it's
+harder to keep track all the active builds. Instead, attempt
```

Comment:
```suggestion
Merging all the changes from upstream in one go is risky since it'll be
more difficult to keep track of all the active builds. Instead, attempt
```